bentoml/yatai/deployment/sagemaker/serve

```diff
@@ -32,7 +28,11 @@ def sigterm_handler(nginx_pid, gunicorn_pid):
     sys.exit(0)
 
 
-def _serve():
+@inject
+def _serve(
+    bento_server_timeout: int = Provide[BentoMLContainer.config.api_server.timeout],
+    bento_server_workers: int = Provide[BentoMLContainer.api_server_workers],
```

Comment:
We can also remove the CPU core related calculation in `BentoMLContainer.api_server_workers` right?

Reply:
We we still want to maintain the behavior to automatically determine works if workers is set to `None`?